src/LabelsTrait.php

```diff
@@ -14,23 +14,51 @@
 
 trait LabelsTrait
 {
+    /**
+     * Returns domain labels
```

Comment:
Returns labels from anything, not just domains. I'd prefer to replace `domain` with `input` here.

Reply:
I did not change the names but yes domain sound good to me I'll update `input` to `domain` everywhere

Reply:
👍